wp-content/themes/eighshift-task/src/blocks/layout/header/_header-navigation.scss

```diff
@@ -0,0 +1,25 @@
+.header-navigation {
+  @extend %list-reset;
+  display: flex;
+  align-items: center;
+  margin: 0 -$base-container-spacing;
```

Comment:
$base-container-spacing is not a part of this component put it inside a map for this component

Reply:
and all other variables in this and any other components